docs/OV_Runtime_UG/supported_plugins/GNA.md

```diff
@@ -338,11 +349,41 @@ and *W* is limited to 87 when there are 64 input channels.
 
 .. note:: 
 
-   The above limitations only apply to the new hardware 2D convolution operation. When possible, the Intel® GNA 
+   The above limitations only apply to the new hardware 2D convolution operation. For GNA 3.0 When possible, the Intel® GNA
    plugin graph compiler flattens 2D convolutions so that the second generation Intel® GNA 1D convolution operations 
    (without these limitations) may be used. The plugin will also flatten 2D convolutions regardless of the sizes if GNA 2.0 
    compilation target is selected (see below).
 
+
+Support for Convolutions since GNA 3.5
+--------------------------------------------------------------------------------------------------------------------------------------
+
+Staring from The Intel® GNA 3.5 convolution 1D are handled in different way than in GNA 3.0. Convolution have following limitations:
```

Comment:
```suggestion
Staring from the Intel® GNA 3.5, 1D convolutions are handled in a different way than in the GNA 3.0. Convolutions have following limitations:
```

Reply:
Adding "the" makes the English sound pretty awkward.  "GNA" is a name.  This is like saying "the Mike".  The other changes are OK. But PLEASE change "Staring" to "starting".